test/demo-contracts/DAIMock.sol

```diff
@@ -22,6 +22,14 @@ contract DAIMock {
         return dec;
     }
 
+    function name() public returns (string memory) {
+        return "DAI Coin";
```

Comment:
You might want to use the name as `Dai Stablecoin` same as Etherscan presentation: https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f

Or https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f#readContract#F6

Reply:
Thank you!